### ISSUE TITLE
How to annotate Table Recognition Datasets

### ISSUE THREAD

**Issue by ghazalli** [PaddlePaddle/PaddleOCR#9388]
Hi, how to annotate Table Recognition Datasets? I want to fine tune the pretrained model, google it but not found any annnotation tools, kindly help.

Thankyou

**Comment by WenmuZhou** [comment]
you can use PPOCRLabel

**Comment by Tanmay98** [comment]
@WenmuZhou Can we also use ppocr label for layout analysis model?
Since the output annotated format doesnot match the format mentioned on the github readme for finetuning layout analysis model

**Comment by ghazalli** [comment]
> @WenmuZhou Can we also use ppocr label for layout analysis model? Since the output annotated format doesnot match the format mentioned on the github readme for finetuning layout analysis model

Let me answer based on my experience, you could annotate the dataset using COCO format and label it based on model you want to fine-tune.
